### ISSUE TITLE
Tf Lite conversion error

### ISSUE THREAD

**Issue by imranmomtaz** [tensorflow/tensorflow#39558]
**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Ubuntu 18.04
- TensorFlow installed from (source or binary): Binary
- TensorFlow version (or github SHA if from source): 2.1.0

**Provide the text output from tflite_convert**

```
Some of the operators in the model are not supported by the standard TensorFlow Lite runtime and are not recognized by TensorFlow. If you have a custom implementation for them you can disable this error with --allow_custom_ops, or by setting allow_custom_ops=True when calling tf.lite.TFLiteConverter(). Here is a list of builtin operators you are using: FILL, FULLY_CONNECTED, PACK, RESHAPE, SHAPE, STRIDED_SLICE, TRANSPOSE. Here is a list of operators for which you will need custom implementations: TensorListFromTensor, TensorListReserve, TensorListStack, While.

```

**Standalone code to reproduce the issue** 
import tensorflow as tf
modelIMU = tf.keras.Sequential()
# inputs: A 3D tensor, with shape [batch, timesteps, feature].
modelIMU.add(GRU(3, input_shape=(2,6,),return_sequences=True))
modelIMU.add(GRU(3))
modelIMU.add(Dense(6))
modelIMU.compile(loss='mean_squared_error', optimizer='adam')
print(modelIMU.summary())

converter = tf.lite.TFLiteConverter.from_saved_model('modelIMU')
converter.optimizations = [tf.lite.Optimize.DEFAULT]
converter.target_spec.supported_ops = [tf.lite.OpsSet.TFLITE_BUILTINS,
                                       tf.lite.OpsSet.SELECT_TF_OPS]
quantized_model = converter.convert()
open("modelIMU.tflite", "wb").write(quantized_model)
Also, please include a link to a GraphDef or the model if possible.

**Any other info / logs**

Include any logs or source code that would be helpful to diagnose the problem.
If including tracebacks, please include the full traceback. Large logs and files
should be attached.

**Comment by Saduf2019** [comment]
@imranmomtaz 
Could you please refer to below issues with similar error and let us know if it helps.
#33490 #29117 #33393 #31844 #[link](https://stackoverflow.com/questions/58069572/workaround-for-lack-of-broadcast-in-tflite) #37134 #32206 #35008 #35818 #33515  #33059

**Comment by imranmomtaz** [comment]
Hello, I was able to solve the problem with the help of https://github.com/tensorflow/tensorflow/issues/34266#issuecomment-554376129. Thanks.
